src/services/api/errors.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
We should create a cleanup issue/epic regarding eslint.

Reply:
I made an issue for this specifically but yes, I would like to do that in December.

Reply:
https://github.com/FixMyBerlin/fixmy.frontend/issues/645